.github/workflows/build-ci.yml

```diff
@@ -32,6 +32,8 @@ jobs:
         run: |
           cat .github/env/${{ runner.os }}/gradle.properties >> gradle.properties
           echo >> gradle.properties
+          .github/scripts/print-gradle-workers-max.sh >> gradle.properties
+          echo >> gradle.properties
```

Comment:
The block below is almost the same.  Too hard to factor the common parts out?

Reply:
I've taken your advice and consolidated it down to a single script.